frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/BuilderFields.tsx

```diff
@@ -29,9 +30,17 @@ export const BuilderFields = ({
   const stateData = useFieldBuilderStore(stateDataSelector)
 
   const { handleBuilderClick } = useCreatePageSidebar()
-  const {
-    deleteFieldModalDisclosure: { onOpen: onDeleteModalOpen },
-  } = useBuilderAndDesignContext()
+
+  const activeFieldNumber =
```

Comment:
We probably want to change the variable name here to not get confused with the question number
```suggestion
  const activeFieldID =
```

Reply:
Oh yes thanks Tim! [c991e3d](https://github.com/opengovsg/FormSG/pull/6303/commits/c991e3dbf4daf3915bb7fd49df8a5c27ce41f2fd)